motr/idx.c

```diff
@@ -150,43 +156,83 @@ static int idx_op_init(struct m0_idx *idx, int opcode,
 	m0_op_idx_bob_init(oi);
 	m0_ast_rc_bob_init(&oi->oi_ar);
 
+	if (ENABLE_DTM0 && M0_IN(op->op_code, (M0_IC_PUT, M0_IC_DEL))) {
```

Comment:
How do we make sure this op has only one pair of KV?
Or where is the checking for this?

Reply:
We check this during initialization of DIX operation, no need to double check.